MobileBuy/sample/src/main/java/com/shopify/sample/application/SampleApplication.java

```diff
@@ -103,6 +114,12 @@ public void failure(RetrofitError error) {
                 Toast.makeText(SampleApplication.this, R.string.shop_error, Toast.LENGTH_LONG).show();
             }
         });
+
+        // Enable Android Pay if we have a public key
+        String androidPayPublicKey = BuildConfig.ANDROID_PAY_PUBLIC_KEY;
+        if (!TextUtils.isEmpty(androidPayPublicKey)) {
+            buyClient.enableAndroidPay(androidPayPublicKey);
```

Comment:
The Android Pay key is optional, it is nice to keep the factory parameters for mandatory options only.